src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -600,6 +602,24 @@ public static void GetFullPath_Windows_StrangeButLegalPaths()
         [InlineData(@"\\.\C:\Foo2\..")]
         public static void GetFullPath_Windows_ValidExtendedPaths(string path)
         {
+            if (PathFeatures.IsUsingLegacyPathNormalization())
+            {
+                // Legacy Path doesn't support any of these paths.
```

Comment:
I've needed something like this more than a few times now. Usually you can use a ThrowsAny on the shared parent of the Exceptions (usually ArgumentException), but in some cases (this one) you have to do the big ugly try/catch.

I don't think I've ever been in a situation where i needed it to expect one of more than 2 exceptions.